test/fuzz/tuf/fuzz_tuff_test.go

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package tuf
+
+import (
+	"testing"
+
+	"github.com/sigstore/sigstore/pkg/tuf"
+)
+
+func FuzzTestAddKey(f *testing.F) {
```

Comment:
Will this test ever fail? AddKey adds a value to a map without verification of the structure of `publicKey`. I think this is testing a map insertion if so.